stable/redis-ha/templates/redis-role.yaml

```diff
@@ -1,20 +1,20 @@
-{{- if .Values.rbac.create -}}
```

Comment:
This was correct.

Reply:
Done

Reply:
@bzumhagen each line in this file is prepended with a `-`. Assuming this was accidental?